enhancements/sig-policy/74-order-policy-execution/README.md

```diff
@@ -0,0 +1,250 @@
+# Guarantee the Ordering of Policy Template Processing
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [website](https://github.com/open-cluster-management-io/open-cluster-management-io.github.io/)
+
+## Summary
+
+Multiple policy templates are often wrapped in one policy, but all this does is create
+all of the policy templates at the same time. In any use case where a policy template 
+depends on another policy being processed before it, the only way to guarantee the policy
+templates are processed in the correct order is to split them into separate policies and
+create them in the desired order. We propose allowing users to specify conditions that
+must be met in order to evaluate a policy template, which would allow the governance
+framework to automatically handle cases where policy templates need to be processed in
+order.
+
+## Motivation
+
+Currently, users have to split up policy templates that depend on others into multiple
+policies and manually create them in order. This is more resource intensive as users
+can have to create duplicate policies and more time consuming if users have to manually
+manage policy processing order. If a user elects to group the templates in one policy,
+there is no guarantee the templates will be processed in order, which could leave
+the policy in a bad state if a required resource has not yet been processed. For example,
+when using a policy to install an operator, if some objects are created before others it
+can cause the operator to not be installed properly. Allowing users to specify policy
+dependencies will alleviate these issues.
+
+### Goals
+
+1. Policy template dependencies: allow users to automatically process policy templates
+in a desired order and guarantee that this order will be stable
+2. Opt-in: any extra fields that are used to specify policy ordering should be optional,
+and policies that do not contain these fields should function as they did before in order
+to avoid issues with older policies
+
+### Non-Goals
+
+1. Status message matching: we plan to only allow users to conditionally process policy
+templates based on compliance state, so users will not be able to specify that they want a
+policy to be processed if another policy has a specific status message
+
+## Proposal
+
+We would add an optional new `dependencies` field to the spec on the configuration 
+policy level as well as an `extraDependencies` field on the template level to
```

Comment:
1. New `dependencies` field is only for configuration policy ? or all types policy ?  Please clarify.
2. Will be implemented these two new fields `dependencies` and `extraDependencies` on UI policy creation wizard ?  Just like 2.6  deletion objects feature (Prune Object Behavior)

Reply:
1. It will be on the Policy type, not any of the specific policy types (like ConfigurationPolicy, CertificatePolicy). Thanks for catching that this wasn't clear here.
2. Yes, we plan to add it to the UI